webpack.dev.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+const merge = require('webpack-merge');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const common = require('./webpack.common');
+
+module.exports = merge(common, {
+  mode: 'development',
+  output: {
+    filename: '[name].bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+  plugins: [
+    new HtmlWebpackPlugin({
+      template: './src/template.html',
+    }),
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.scss$/,
+        use: [
+          'style-loader',
+          'css-loader',
+          'sass-loader',
+        ],
+      },
+    ],
+  },
+});
```

Comment:
## Improve the webpack configuration to have the dist directory.
- [ ] The project expects to have a `dist` directory that contains the chunked out minimised code to show on the page. That ought to have been the main purpose of the webpack in this project. As a result, the `dist` dir is expected to have:
- [ ] A main file named `index.html`
- [ ] A `main.js` file that is built by webpack
- [ ] The `index.html` links the `main.js` file as shown below:

```javascript
const myModule = (parentNode) => {
  const node = document.createElement("div");
  // Add whichever sub-nodes, ids and style is required

  parentNode.appendChild(node);
};

export { myModule as default };
```

### Note: 
_The `npm run build` command is not functioning as expected. Hence the detailed review up here_ ☝🏾